cumulusci/tasks/marketing_cloud/mc_constants.py

```diff
@@ -1 +1,5 @@
 MC_API_VERSION = "v2"
+
+# We *do not* want to derive this URL from `rest_instance_url` because
+# it could potentially change in the future.
+AUTH_URI = "https://{}.auth.marketingcloudapis.com"
```

Comment:
Crazy idea but have thought about supporting constants at the cumulusci.yml level for a project?